src/datasets.js

```diff
@@ -10,7 +10,8 @@ import {
 } from './utils'; // eslint-disable-line import/no-duplicates
 import * as Utils from './utils'; // eslint-disable-line import/no-duplicates
```

Comment:
It is because of the stubs in tests. The two methods that are called  by `utils.*` are mocked the rest is not and I would find quite irritating to write all the `pluckData` with the `utils` prefix.

Reply:
Could we have one import statement of `utils` and destructure from it below, rather than two import statements then?